src/Stripe.Tests.XUnit/BaseStripeTest.cs

```diff
@@ -0,0 +1,161 @@
+using System;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using Xunit;
+
+namespace Stripe.Tests.Xunit
+{
+    public class BaseStripeTest : IDisposable
+    {
+        /// <value>Minimum required version of stripe-mock</value>
+        /// <remarks>
+        /// If you bump this, please don't forget to bump STRIPE_MOCK_VERSION in appveyor.yml.
+        /// </remarks>
+        private const string MockMinimumVersion = "0.19.0";
+
+        private static string Port;
+
+        private string OrigApiBase;
```

Comment:
Orig => Original ? (stripe-dotnet does not really abbreviate variable names, let's not start that trend)